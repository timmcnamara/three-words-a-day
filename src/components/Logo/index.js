import React from "react";
import styled from "styled-components";
import { light } from "../../styles/styleGuide";

const Icon = styled.svg`
  fill: ${light};
  width: 48px;
  height: 48px;
`;

const Logo = () => (
  <Icon viewBox="0 0 20 20">
    <path d="M10 15h8c1 0 2-1 2-2V3c0-1-1-2-2-2H2C1 1 0 2 0 3v10c0 1 1 2 2 2h4v4l4-4zM5 7h2v2H5V7zm4 0h2v2H9V7zm4 0h2v2h-2V7z" />
  </Icon>
);

export default Logo;
