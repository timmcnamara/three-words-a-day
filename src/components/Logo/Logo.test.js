import { render, cleanup } from "react-testing-library";
import React from "react";
import Logo from ".";
import "jest-dom/extend-expect";

afterEach(cleanup);

describe("<Logo/>", () => {
  it("renders navigation component", () => {
    const { asFragment } = render(<Logo />);

    expect(asFragment()).toMatchSnapshot();
  });
});
