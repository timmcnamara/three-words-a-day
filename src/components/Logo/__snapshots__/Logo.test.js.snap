// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo/> renders navigation component 1`] = `
<DocumentFragment>
  <svg
    class="sc-bdVaJa irjXTH"
    viewBox="0 0 20 20"
  >
    <path
      d="M10 15h8c1 0 2-1 2-2V3c0-1-1-2-2-2H2C1 1 0 2 0 3v10c0 1 1 2 2 2h4v4l4-4zM5 7h2v2H5V7zm4 0h2v2H9V7zm4 0h2v2h-2V7z"
    />
  </svg>
</DocumentFragment>
`;
